collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,151 @@
+
+While v0.8.0 delivers significant advancements, we're already working on v0.9.0! 
+
+<!-- Checkout [Meshery’s roadmap](https://github.com/meshery/meshery/blob/master/ROADMAP.md) to peek into what’s yet to come. -->
+
+We are already hard at work on [v0.9.0 Roadmap](https://discuss.layer5.io/t/meshery-v0-9-roadmap/6296)! See [v0.9.0 Milestone](https://github.com/meshery/meshery/milestone/5) in `meshery/meshery` for more details and the [Meshery Architecture deck](https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#) for additional visual designs and functional specifications. Stay tuned for more updates.
+
+* **Adapters & Performance Management:**  Evolving `meshes.proto` to `adapters.proto` and externalizing Nighthawk for improved performance testing.
+* **Basic Adaptive Load Controller:**  Introducing adaptive testing capabilities for more dynamic performance analysis.
+* **Custom Telemetry Provider:** Enabling user-defined dashboards and metrics for greater flexibility.
+
+<details>
+<summary>Roadmap Details</summary>
+
+<h3>Adapters & Performance Management</h3>
+
+<ol>
+<li><a href="https://github.com/meshery/meshery/issues/9968">Evolve meshes.proto to adapters.proto</a></li>
+<li> Externalize Nighthawk (<a href="https://docs.google.com/document/d/1Qy_BpHXibTvL9daLTc5L4fjmr1F051ya0UYeghNUq_I/edit?usp=sharing">spec</a>)</li>
+    <li>Nighthawk as a Meshery Adapter (`meshery-nighthawk`)</li>
+    <li>Server: Lifecycle Management of `meshery-nighthawk` in Meshery Server</li>
+    <li>CLI: `mesheryctl perf` support for load generator / adapter selection</li>
+    <li>UI: `meshery-nighthawk` adapter chip</li>
+</ol>
+
+<h3>Basic Adaptive Load Controller</h3>
+
+**Basic Adaptive Load Controller** ([spec](https://docs.google.com/presentation/d/1arewKYZuhkv7NGRZKx0w-7bQdS-ho-YSYJF0X32qfHo/edit#slide=id.gfbc0c4fb3f_0_8)) (_v0.9.0_)
+<ul>
+<li>Exposure of “Adaptive Test” in Meshery UI.</li>
+<li>Meshery Server endpoints.</li>
+<li>Storage of n result sets in Provider for a given Performance Profile.</li>
+</ul>
+
+<h3>Custom Telemetry Provider</h3>
+
+<ul>
+<li>Multiple Telemetry Providers and Custom Boards (<a href="https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g1044af767ce_5_2">spec</a>)</li>
+<li>User-defined Dashboards and Metrics (<a href="https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.gcb74201a11_0_119">spec</a>)</li>
+</ul>
+  
+</details>
+
+**Dive deeper into the details and explore the full list of changes in the [Meshery Documentation](https://docs.meshery.io/docs/project/releases). Stay tuned for updates on [v0.9.0](https://discuss.layer5.io/t/meshery-v0-9-roadmap/6296)!**
+
+Try v0.8.0 in the [Meshery Playground](https://play.meshery.io) or [download](https://docs.meshery.io/installation) the latest release to get started. We look forward to your feedback and contributions!
```

Comment:
Should Meshery Playground link be https://playground.meshery.io or https://kanvas.new? Or it is deliberately at https://play.meshery.io for the onboarding flow?

Reply:
Great question. Yes, you're right that it is listed as play.meshery.io for the overview and explantation of what it is, why you need to sign in, and so on prior to pointing users directly to a sign-up/sign-in page.